templates/_layout.html

```diff
@@ -12,11 +12,13 @@
   {% include '_app_scripts.html' %}
 </head>
 <body>
-  <p class="bg-text">PassION<sup>BETA</sup></p>
+  <span class="fork-on-github"><a href="https://github.com/skooda/passIon">Fork me on GitHub</a></span>
```

Comment:
`https://github.com/heureka/passIon`

Reply:
vidis, to me mohlo trknout, ze je to potreba fixnout :D